genet/max_stable_set.py

```diff
@@ -19,6 +20,10 @@ def has_attrib(attrib_value):
     return True
 
 
+def process_model_result(result_list: List[str]):
```

Comment:
I am not sure about the name of this function. Is it processing model results? From what I can see, it's transforming stop names. Or is it more generic than that? Can we come up with a more descriptive name?